man/get_cdc_data.Rd

```diff
@@ -0,0 +1,36 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/get_cdc_data.R
+\name{get_cdc_data}
+\alias{get_cdc_data}
+\title{Retrieve CDC growth chart metadataset}
+\usage{
+get_cdc_data(type)
+}
+\arguments{
+\item{type}{Type of Chart
+
+A character is expected amongst: "height", "weight", "bmi"}
```

Comment:
need to re-run `devtools::document()`

Reply:
Also not sure this documentation is looking so clean when rendered - see below. i think we could remove the `double` references and add more to the bottom line such as `This metadata is sourced from [CDC website](https://www.cdc.gov/growthcharts/percentile_data_files.htm).`

![image](https://github.com/pharmaverse/admiralpeds/assets/82581364/a06ab71a-9dfa-4e9d-8ef6-6561a67223df)